### DISCUSSION TITLE
Install pdf-tools on Mac OS with macports

### DISCUSSION THREAD

**Discussion by gpagnon** [vedang/pdf-tools#226] [General / FAQ]
Hi,
I am trying to use the pdf layer on spacemacs on a mac, but it seems that the installation of pdf-tools through spacemacs is not working, as when I compile a latex file and view the output I see the pdf code instead of the rendered document.

I have seen on this page instructions to install pdf-tool on a mac using brew. However, I am using macports for a lot of software and I would not like to also install homebrew.   Is there a way to install pdf-tools using the macports dependencies?

thanks for any suggestions!
giuseppe

**Comment by velppa** [comment]
I stumbled upon the same as I'm migrating from Homebrew to MacPorts.  After I uninstalled Homebrew, pdf-tools used MacPorts to build the epdfinfo program.  Just make sure that Emacs has `port` in its PATH - `(executable-find "port")` should return the correct path.
